src/composer.cpp

```diff
@@ -117,19 +153,199 @@ void Binatr::applyNAppend(const Syntop& op, Bitwriter* bits) const
        throw std::string("Binary translator: null writer pointer.");
     bits->startInstruction();
     size_t argnum = 0;
+    std::vector<size_t> reordering;
+    if (!m_reordering.size())
+    {
+        reordering.reserve(op.size());
+        for (size_t rodr = 0; rodr < op.size(); rodr++) reordering.push_back(rodr);
+    }
+    else
+        reordering = m_reordering;
+
+#if defined(_WIN32)                                      //TODO(ch): IMPORTANT. Destroy it. Devastate. Annihilate. 
```

Comment:
if it's Intel only, move it to Backend. Or make separate composer for Intel. Overall, I don't quite understand (and maybe this is why don't quite like) the current separation between composer and backend. But if I do this decomposition, please, make it in a real way, do not introduce many platform-specific magic numbers into the cross-platform class

Reply:
Done.